configs/bld-win64.cmd.user_data

```diff
@@ -0,0 +1,8 @@
+<script>
+sc delete puppet
+cscript.exe "C:\ProgramData\Puppetlabs\puppet\var\puppettize_TEMP.vbs"
+cd "C:\Program Files (x86)\Puppet Labs\Puppet\bin"
+cmd /c puppet agent --test --environment=mcornmesser --server=releng-puppet2.srv.releng.scl3.mozilla.com
+schtasks /delete /TN RUNPUPPET /F
+shutdown /s /t 0 /c "userdata run complete" /f /d p:4:1
+</script>
```

Comment:
regular batch scripts don't exit if a command fails (there is nothing like `set -e` like in bash) - do we know if one of the earlier steps fails, whether subsequent steps will run? In other words, is this copied to target machine and then executed? If so, maybe we need checking of %errorlevel%?